app/models/state_file_base_intake.rb

```diff
@@ -124,7 +124,7 @@ def synchronize_df_w2s_to_database
         employer_name: direct_file_w2.EmployerName,
         employee_name: direct_file_w2.EmployeeNm,
         employee_ssn: direct_file_w2.EmployeeSSN,
-        employer_state_id_num: direct_file_w2.EmployerStateIdNum,
+        employer_state_id_num: direct_file_w2.EmployerStateIdNum&.delete('-'),
```

Comment:
`miranda_1099r` fixture has a dash in the `EmployerStateIdNum`. According to schema i think this is a valid character. However, since we validate this field (`employer_state_id_num`) on save to the database, we should remove any dashes. Hopefully this is the only dash that I may expect, not an em dash...